### ISSUE TITLE
Error "Object argument required."

### ISSUE THREAD

**Issue by Sector95** [TsvetanMilanov/serverless-base-path-mapping#2]
In one of the more recent versions of the Serverless Framework, it appears that something changed that causes this plugin to throw an error.

Error shows up in Serverless version 1.26.1

I'll see if I can dig in some more when I get some free time.

**Comment by TsvetanMilanov** [comment]
@Sector95 Thank you for reporting this issue. Can you execute the deploy command with `SLS_DEBUG=*` and share the output here?

**Comment by Sector95** [comment]
Here's the stack trace:
```
Error: Object argument required.
    at applyDefaults (/usr/local/lib/node_modules/serverless/node_modules/object-hash/index.js:79:11)
    at objectHash (/usr/local/lib/node_modules/serverless/node_modules/object-hash/index.js:31:13)
    at Function.exports.sha1 (/usr/local/lib/node_modules/serverless/node_modules/object-hash/index.js:44:10)
    at AwsProvider.request (/usr/local/lib/node_modules/serverless/lib/plugins/aws/provider/awsProvider.js:202:35)
    at CreateBasePathMappingService._getRestApiInfo (/home/circleci/project/node_modules/serverless-base-path-mapping/create-base-path-mapping-service.js:55:25)
    at CreateBasePathMappingService._createBasePathMapping (/home/circleci/project/node_modules/serverless-base-path-mapping/create-base-path-mapping-service.js:42:15)
    at _getMappingInfo.then.mappingInfo (/home/circleci/project/node_modules/serverless-base-path-mapping/create-base-path-mapping-service.js:22:18)
From previous event:
    at CreateBasePathMappingService.createBasePathMapping (/home/circleci/project/node_modules/serverless-base-path-mapping/create-base-path-mapping-service.js:19:5)
    at BasePathMappingPlugin._beforeDeploy (/home/circleci/project/node_modules/serverless-base-path-mapping/index.js:17:39)
    at BbPromise.reduce (/usr/local/lib/node_modules/serverless/lib/classes/PluginManager.js:372:55)
From previous event:
    at PluginManager.invoke (/usr/local/lib/node_modules/serverless/lib/classes/PluginManager.js:372:22)
    at PluginManager.run (/usr/local/lib/node_modules/serverless/lib/classes/PluginManager.js:403:17)
    at variables.populateService.then (/usr/local/lib/node_modules/serverless/lib/Serverless.js:102:33)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
From previous event:
    at Serverless.run (/usr/local/lib/node_modules/serverless/lib/Serverless.js:89:74)
    at serverless.init.then (/usr/local/lib/node_modules/serverless/bin/serverless:42:50)
    at <anonymous>
```

**Comment by TsvetanMilanov** [comment]
@Sector95 Thank you for the stack trace. I found the problem. The fix is released in version 0.1.1
